plugin/trino-delta-lake/src/main/java/io/trino/plugin/deltalake/transactionlog/writer/S3NativeTransactionLogSynchronizer.java

```diff
@@ -61,22 +63,48 @@ public class S3NativeTransactionLogSynchronizer
 
     private final TrinoFileSystemFactory fileSystemFactory;
     private final JsonCodec<LockFileContents> lockFileContentsJsonCodec;
+    private final boolean exclusiveWriteSupported;
```

Comment:
nit: write or create? minor difference, but I guess lets be consistent in the naming